### ISSUE TITLE
[4.0.0-beta.15] no-margin doesn't work for ion-label in an ion-item-divider

### ISSUE THREAD

**Issue by jeroenklein** [ionic-team/ionic#16245]
# Bug Report

**Ionic Info**
```
Ionic:

   Ionic Framework               : @ionic/angular 4.0.0-beta.15
```

**Describe the Bug**
After upgrade from ```@ionic/angular@4.0.0-beta.13``` to ```@ionic/angular@4.0.0-beta.15``` the CSS utility attribute "no-margin" doesn't work on the ```ion-label``` in an ```ion-item-divider```.

**Steps to Reproduce**
Add
```
<ion-item-divider color="primary">
 <ion-label no-margin>
  <h1>Hello World</h1>
 </ion-label>
</ion-item-divider>
```
and the margins will not be removed by ```no-margin```.

**Related Code**
https://codepen.io/anon/pen/GwJaXd

**Expected Behavior**
Working sample with ```@ionic/angular@4.0.0-beta.13```: 
https://codepen.io/anon/pen/ZmGNMY

**Comment by MichaelMalony** [comment]
Having the same problem inside <ion-item>. Definitely new to beta 15.

**Comment by jeroenklein** [comment]
The issue is fixed in `@ionic/angular@4.0.0-beta.17`. See https://codepen.io/anon/pen/YRMjPd
